package.json

```diff
@@ -25,6 +25,7 @@
     "e2e:create-test-user": "ts-node bin/e2e/create-test-user.ts",
     "e2e:remove-test-user": "ts-node bin/e2e/remove-test-user.ts",
     "e2e:create-status-check": "ts-node bin/e2e/create-e2e-status-check.ts",
+    "generate-connect-profile-client": "docker run --rm -v ${PWD}/:/local openapitools/openapi-generator-cli generate --input-spec /local/openapi.yml --generator-name typescript-axios --additional-properties supportsES6=true --output /local/connect-profile-client",
```

Comment:
This command will be used each time we change the `openapi.yml` file to generate a new client.

Reply:
Could we name the `openapi.yml` `connect-profile-openapi.yml`?

Reply:
done